src/nodes/dir.rs

```diff
@@ -171,7 +264,12 @@ impl Node for Dir {
         reply.add(state.parent, 1, fuse::FileType::Directory, "..");
         let mut pos = 2;
 
-        // TODO(jmmv): Handle user-provided mappings before underlying entries.
+        for (name, dirent) in &state.children {
```

Comment:
This section could use a comment. It took me multiple readings to understand why you can't do this in the loop below.

Reply:
Yep, done.